src/main/scala/is/hail/expr/Type.scala

```diff
@@ -133,7 +133,7 @@ sealed abstract class Type {
 
   def str(a: Annotation): String = if (a == null) "NA" else a.toString
 
-  def strVCF(a: Annotation): String = if (a == null) "." else a.toString
+  def strVCF(a: Annotation): String = if (a == null) "." else str(a)
```

Comment:
can you move strVCF to ExportVCF?  It's a specific application that doesn't quite belong on Type